lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -655,6 +655,7 @@ export class UserAgentApplication {
               // App uses idToken to send to api endpoints
               // Default scope is tracked as clientId to store this token
               this.logger.verbose("renewing idToken");
+              this.silentLogin = true;
```

Comment:
Can you explain what this option does?

Reply:
Essentially it sets the fact that we are logging in (requesting only an id token) silently. It isn't a great design, and we are looking at refactoring this to take this out, but for now it exists so I am leveraging it here.

Reply:
Looks okay but the name bugs me. ATS would have to be ideally silentLogin - but looks like we are adding it for cases of adal idToken -> ATS route only previously. Using the same for an idToken renewal is confusing, Having said that I cannot yet think of a better option. Let's discuss this in person once.

Reply:
I think this variable doesn't even need to be there, but we can discuss that another time. For now we should approve and merge this.

Reply:
That I agree. Maybe no sense in renaming it now.